Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
-        }
+            // Loop twice. Can be changed by altering the number that "i" is less than.
+            for (int i = 0; i < 2; i++)
+            {
+                // Get name, age, weight, and height from the user.
+                Console.Write("Please enter your surname: ");
+                string name = Console.ReadLine();
+                Console.Write("Please enter your age: ");
+                var age = Console.ReadLine();
+                Console.Write("Please enter your weight (in kg): ");
+                var weight = Console.ReadLine();
+                Console.Write("Please enter your height (in m): ");
+                var height = Console.ReadLine();
+                // Set the user data in a sentence and then write it to the console.
+                string sentence = (name + " is " + age + " " + "years old, his weight is " + weight + "kg and his height is " + height + "m.");
```

Comment:
Mixing styles really isn't a good thing. This can confuse you and other readers when reading (your own) code. Therefor I'd request you to either use var or explict built-in types (like string, int, etc.) only and if you ever break your chosen style, do it only if you have a valid reason to do so.

Reply:
I just fixed it and pushed it. Do I need a new pull request or will it automatically update here as well after the push from VS Studio?